Development/src/theme/ThemeContext.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { ThemeProvider } from '@material-ui/styles';
+import { createMuiTheme } from '@material-ui/core/styles';
+import { lightBlue } from '@material-ui/core/colors';
+import Cookies from 'universal-cookie';
+
+const cookies = new Cookies();
+
+const ThemeToggleContext = React.createContext('light');
+export const useTheme = () => React.useContext(ThemeToggleContext);
+
+export const AppThemeProvider = ({ children }) => {
+    const [themeState, setThemeState] = React.useState({
+        mode: cookies.get('theme'),
+    });
+    if (themeState.mode === undefined) setThemeState({ mode: 'light' });
+
+    const theme = createMuiTheme({
+        palette: {
+            primary: lightBlue,
+            secondary: lightBlue,
```

Comment:
Probably seems a bit wrong to use the same colour for primary and secondary?

Reply:
The secondary colour is used in the same way as the primary in dark mode. I wanted to keep elements light blue, however we could use a different colour when in dark mode.

Reply:
Let's discuss, one more shade in the light mode might not be a bad thing